glide.yaml

```diff
@@ -34,13 +34,24 @@ import:
 - package: github.com/giantswarm/certificatetpr
   version: fa554d09f0137a91aa6ef007b92e170b31dbefd2
 - package: github.com/giantswarm/k8scloudconfig
-  version: b31c9407200797ca4be735c3a73ec51ae34b31fd
+  version: f194c3c15b3218dd5e21606cd5dd190818a64ab3
 - package: github.com/golang/protobuf
   version: 8ee79997227bf9b34611aee7946ae64735e6fd93
   subpackages:
   - jsonpb
   - proto
   - sortkeys
+# The version of cobra and pflag are indicated by kubernetes/go-client, but using their current
+# versions results in the following error:
+# vendor/github.com/spf13/cobra/command.go:1092: c.lflags.SortFlags undefined (type *pflag.FlagSet has no field or method SortFlags)
+# vendor/github.com/spf13/cobra/command.go:1092: c.Flags().SortFlags undefined (type *pflag.FlagSet has no field or method SortFlags)
+# vendor/github.com/spf13/cobra/command.go:1240: c.parentsPflags.SortFlags undefined (type *pflag.FlagSet has no field or method SortFlags)
+#
+# TODO(nhlfr): Keep pining the versions of cobra and pflag or propose the fix to client-go upstream.
```

Comment:
The issue is finally a little bit different from what I wrote yesterday.

client-go has a pinned version of spf13/pflag which conflicts with the newest cobra we get.
Of course I will try to propose updating pflag in client-go, I just wanted you to be aware of the situation.